styles/styles.css

```diff
@@ -580,19 +580,39 @@ button {
 }
 
 .button-container a.primary,
-.button-container a.secondary {
-  box-sizing: border-box;
-  height: 48px;
-  border-radius: 40px;
-  padding: 11px var(--spacer-layout-02);
+ .button-container a.secondary {
+   box-sizing: border-box;
+   height: 48px;
+   border-radius: 40px;
+   padding: 11px var(--spacer-layout-02);
+ }
+
+.button-container a.tertiary {
+  height: 20px;
+  line-height: 20px;
+  align-items: end;
 }
 
 /** Button icons **/
+.article-content .icon > svg {
+  width: 32px;
+  height: 32px;
+}
+
+.article-content a + .icon > svg {
+  width: 20px;
+  height: 20px;
+}
```

Comment:
This may not be needed anymore since it existed prior to global button styling.

Reply:
I saw some other "article" specific style as well, but I couldn't find any "article' pages to test out, so I left it alone as well 😃